service/src/main/java/uk/gov/hmcts/reform/fpl/service/docmosis/CaseSubmissionGenerationService.java

```diff
@@ -552,172 +609,189 @@ private String getDefaultIfNullOrEmpty(final Optional<String> value) {
         return value.filter(StringUtils::isNotEmpty).orElse(DEFAULT_STRING);
     }
 
-    private String getChildLivingSituation(final ChildParty child, final boolean isConfidential) {
+    private String getChildLivingSituation(final ChildParty child, final boolean isConfidential,
+                                           Language applicationLanguage) {
         StringBuilder stringBuilder = new StringBuilder();
         if (StringUtils.isNotEmpty(child.getLivingSituation())) {
             stringBuilder.append(child.getLivingSituation());
 
             if (isConfidential) {
-                stringBuilder.append(NEW_LINE).append(CONFIDENTIAL);
+                stringBuilder.append(NEW_LINE).append(getConfidential(applicationLanguage));
             } else if (isNotEmpty(child.getAddress())) {
                 stringBuilder.append(NEW_LINE).append(child.getAddress().getAddressAsString(NEW_LINE));
             }
 
             stringBuilder.append(endsWith(stringBuilder.toString(), NEW_LINE) ? "" : NEW_LINE);
-            formatChildLivingSituationDisplay(child, stringBuilder);
+            formatChildLivingSituationDisplay(child, stringBuilder, applicationLanguage);
         }
 
         return StringUtils.isNotEmpty(stringBuilder.toString()) ? stringBuilder.toString().trim() : DEFAULT_STRING;
     }
 
-    private void formatChildLivingSituationDisplay(final ChildParty child, final StringBuilder sb) {
+    private void formatChildLivingSituationDisplay(final ChildParty child, final StringBuilder sb,
+                                                   Language applicationLanguage) {
         switch (fromString(child.getLivingSituation())) {
             case HOSPITAL_SOON_TO_BE_DISCHARGED:
                 if (child.getDischargeDate() != null) {
-                    sb.append("Discharge date: ").append(formatDateDisplay(child.getDischargeDate()));
+                    sb.append("Discharge date: ")
+                        .append(formatDateDisplay(child.getDischargeDate(), applicationLanguage));
                 }
                 break;
             case REMOVED_BY_POLICE_POWER_ENDS:
                 if (child.getDatePowersEnd() != null) {
-                    sb.append("Date powers end: ").append(formatDateDisplay(child.getDatePowersEnd()));
+                    sb.append("Date powers end: ")
```

Comment:
Is this getting translated?

Reply:
To be translated in [DFPL-255](https://tools.hmcts.net/jira/browse/DFPL-255).